src/index.html

```diff
@@ -5,8 +5,415 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo logo" class="header__image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" data-qa="nav-hover" class="nav__link">Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+
+        <div class="container__card card" data-qa="card">
+
+          <img src="/images/imac.jpeg" alt="logo" class="card__logo">
```

Comment:
There shouldn't be any blanklines between parent and child elements
```suggestion
      <div class="container">
        <div class="container__card card" data-qa="card">
          <img src="/images/imac.jpeg" alt="logo" class="card__logo">
```